tests/helpers/index.js

```diff
@@ -1,18 +1,50 @@
+import { randomBytes } from 'crypto'
 import { KeyManager } from '@mapeo/crypto'
 
 /**
  * @param {string} name
  * @param {Buffer} [namespace] - 32 byte Buffer
  */
-export function createCoreKeyPair(name, namespace = Buffer.alloc(32, 0)) {
-  const { km } = createIdentityKeys()
-  const coreKeyPair = km.getHypercoreKeypair(name, namespace)
+export function createCoreKeyPair(name, namespace = randomBytes(32)) {
+  const { keyManager } = createIdentityKeys()
+  const coreKeyPair = keyManager.getHypercoreKeypair(name, namespace)
   return coreKeyPair
 }
 
 export function createIdentityKeys() {
   const rootKey = KeyManager.generateRootKey()
-  const km = new KeyManager(rootKey)
-  const identityKeyPair = km.getIdentityKeypair()
-  return { rootKey, identityKeyPair, km }
+  const keyManager = new KeyManager(rootKey)
+  const identityKeyPair = keyManager.getIdentityKeypair()
+  const identityId = identityKeyPair.publicKey.toString('hex')
+  return { rootKey, identityId, identityKeyPair, keyManager }
+}
+
+export function replicate(peers) {
+  for (const peer1 of peers) {
+    for (const peer2 of peers) {
+      if (peer1 === peer2) continue
+      const stream1 = peer1.authstore.replicate(true, { live: true })
+      const stream2 = peer2.authstore.replicate(false, { live: true })
+      stream1.pipe(stream2).pipe(stream1)
+    }
+  }
+}
```

Comment:
Won't this duplicate replication?
e.g. two peers A, B then this will do `A.pipe(B).pipe(A)` _and_ `B.pipe(A).pipe(B)`

Reply:
ah that would! fixed in recent commit.